apps/stats-dapp/src/provider/hooks/useProposals.ts

```diff
@@ -605,3 +607,106 @@ export function useVotes(votesReqQuery: VotesQuery): VotesPage {
 
   return votes;
 }
+
+export type TimeRange =
+  | 'all'
+  | 'one-year'
+  | 'six-months'
+  | 'three-months'
+  | 'year-to-date';
+
+export type ProposalsOvertimeTotalCount = {
+  month: number;
+  proposalType: ProposalType | string;
+  totalCount: number;
+}[];
+
+/**
+ * Proposals Overtime Count
+ * @return ProposalsOvertimeTotalCount - Proposal overtime total count data
+ * */
+export function useProposalsOvertimeTotalCount(
+  timeRange: TimeRange
+): Loadable<ProposalsOvertimeTotalCount> {
+  const [proposalsOvertimeCount, setProposalsOvertimeCount] = useState<
+    Loadable<ProposalsOvertimeTotalCount>
+  >({
+    isLoading: true,
+    val: null,
+    isFailed: false,
+  });
+
+  const client = useApolloClient();
+
+  const {
+    blockTime,
+    metaData: { lastProcessBlock },
+  } = useStatsContext();
+
+  const blockRanges: BlockRanges = useMemo(() => {
+    let _blockRanges: BlockRanges = [];
```

Comment:
The `_` before a variable name normally means that the variables isn't in use please remove it